### PR TITLE
Add TLS (attempt 3)

### PR DESCRIPTION
**Pull request by paralainer**
Hey, bitnami finally supported TLS in their container, if you don't mind trying again, I'd like to try with it.
Although I could not reproduce the same error neither locally or on Github Actions.

### CODE REVIEW

File: folsom/src/test/java/com/spotify/folsom/MemcachedServer.java [hunk 1]
```diff
@@ -78,7 +112,30 @@ public MemcachedServer(
         break;
     }
 
-    start(authenticationMode);
+    return container;
+  }
+
+  private FixedHostPortGenericContainer setupTLSContainer() {
+    final FixedHostPortGenericContainer container =
+        new FixedHostPortGenericContainer("bitnami/memcached:" + MEMCACHED_VERSION);
```
Comment (paralainer) [inline folsom/src/test/java/com/spotify/folsom/MemcachedServer.java:120]: Using `bitnami` container here now

File: folsom/src/test/java/com/spotify/folsom/MemcachedServer.java [hunk 2]
```diff
@@ -38,31 +40,63 @@ public enum AuthenticationMode {
       Suppliers.memoize(MemcachedServer::new)::get;
 
   public static int DEFAULT_PORT = 11211;
+
+  private static final String MEMCACHED_VERSION = "1.6.22";
```
Comment (paralainer) [inline folsom/src/test/java/com/spotify/folsom/MemcachedServer.java:44]: Had to bump the version to get SSL support within a container

**Comment by thiagocesarj** [comment]
@paralainer I have just merged a whole bunch of dependency bumps, including testcontainers. These together with your changes might finally make these tests stable. I will be trying to merge again today. :crossed_fingers:

**Review by thiagocesarj — APPROVED** [review]

**Comment by thiagocesarj** [comment]
It's all green! Thank you for the contribution and for patiently fixing all of the tests @paralainer !

**Comment by paralainer** [comment]
Thank you @thiagocesarj for all your support!

**Comment by paralainer** [comment]
@thiagocesarj hey, when do you expect to release 1.18.0?

**Pull request merged**